include/pomerol/DensityMatrix.h

```diff
@@ -29,6 +29,8 @@ class DensityMatrix : public Thermal, public ComputableObject
     const Hamiltonian &H;
     /** A vector of pointers to parts (every part corresponds to a part of the Hamiltonian). */
     std::vector<DensityMatrixPart*> parts;
+    /** A vector of bool: true if the block has not been truncated. */
+    std::vector<bool> block_retained;
```

Comment:
This better be initialized to all true if TruncateBlocks() method was not called.

Reply:
I agree. Updated in 652f7b8